spec/lib/appsignal/cli/diagnose_spec.rb

```diff
@@ -17,6 +17,7 @@
 
       out_stream.string.should include('Gem version')
       out_stream.string.should include('Agent version')
+      out_stream.string.should include('Environment')
```

Comment:
Nitpicking: could you add the actual environment to the assertion here to make sure it's printed correctly, or is there any reason `Appsignal.config.env` is unstable in the tests?